go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -358,6 +361,24 @@ func NewVitessCluster(t *testing.T, name string, cellNames []string, clusterConf
 	return vc
 }
 
+// CleanupDataroot deletes the vtdataroot directory. Since we run multiple tests sequentially in a single CI test shard,
+// we can run out of disk space due to all the leftover artifacts from previous tests.
+func (vc *VitessCluster) CleanupDataroot(t *testing.T) {
+	// This is always set to "true" on GitHub Actions runners:
+	// https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+	ci, ok := os.LookupEnv("CI")
+	if !ok || strings.ToLower(ci) != "true" {
+		// Leave the directory in place to support local debugging.
+		return
+	}
+	dir := vc.ClusterConfig.vtdataroot
+	log.Infof("Deleting vtdataroot %s", dir)
+	err := os.RemoveAll(dir)
+	require.NoError(t, err)
+	err = os.Mkdir(dir, 0755)
```

Comment:
We don't need `other` to have any privileges do we? i.e. I think we should use 750 unless there's some special reason.

Reply:
used `0700`